packages/cluster-ui/src/statementsTable/statementsTable.module.scss

```diff
@@ -24,7 +24,7 @@
   }
 }
 
-.statements-table__col-count, .statements-table__col-retries, .statements-table__col-rows {
+.statements-table__col-count, .statements-table__col-rows, .statements-table__col-rows-read, .statements-table__col-bytes-read , .statements-table__col-max-mem-usage, .statements-tabe__col-network-bytes, .statements-table__col-retries {
```

Comment:
there's a `tabe` -> `table` spelling error here. I will push a small commit to your branch that cleans this up. It's quite annoying to deal with and we should only need to use a single selector here.